Added more override examples

lukewarmtemp opened the pull request:
Fixes #4054 
- Added example for local file overrides
- Added example or Fedora Koji/Bodhi overrides
- Added example for overriding podman based on the version defined in the Fedora updates-testing repo

lukewarmtemp commented:
@travier Do you think I should add more examples?

travier commented on docs/ex-replace.md:
````diff
@@ -82,6 +82,59 @@ Deployments:
 
+In this second example, we will override the `podman` rpm package with the version of the package used in the Fedora *updates-testing* repository. The URL of the repository can be chosen from those listed in the [Fedora Mirror List](https://mirrors.fedoraproject.org/mirrorlist?repo=updates-testing-f38&arch=x86_64). 
+```
+$ cat > /etc/yum.repos.d/updates.repo << EOF
````
There should already be those repo configs in /etc/yum.repos.d/fedora-updates-testing.repo. They only need to be enabled. You can use `sed` to tweak the config to enable them.

travier commented on docs/administrator-handbook.md:
````diff
@@ -191,6 +191,16 @@ both direct HTTP URLs as well as local files:
+
+It is also supported to pull from the Fedora Koji/Bodhi systems:
+```
+$ rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2020-XXXXXXX
````
Let's find real examples for podman as well

travier reviewed (changes requested):
Some comments but looks great! Thanks

lukewarmtemp commented:
- Added real examples for `podman` using Koji and Bodhi
- Used `sed` to set the the `enable` flag to true on the existing fedora-updates-testing repo
- Fixed some spelling mistakes

travier reviewed (approved):
LGTM. Not sure why CI is failing here as this is a docs only PR

lukewarmtemp commented:
/retest-required

HuijingHei commented:
Just one minor comment, the others LGTM. And could you help to squash the commits to one?

HuijingHei commented on docs/administrator-handbook.md:
````diff
@@ -191,6 +191,16 @@ both direct HTTP URLs as well as local files:
 ```
+```
+$ curl https://rpmfind.net/linux/fedora/linux/updates/testing/38/Everything/x86_64/Packages/p/podman-4.5.1-1.fc38.x86_64.rpm --output podman.rpm
+$ rpm-ostree override replace ./podman.rpm
````
This needs admin to run, maybe can update like $ sudo rpm-ostree override replace ./podman.rpm ? And the next 2 commands.

HuijingHei commented:
> LGTM. Not sure why CI is failing here as this is a docs only PR

Looks wired, maybe can skip CI for doc update. https://github.com/openshift/release/pull/40044

HuijingHei reviewed (approved):
Nice work! LGTM

lukewarmtemp commented:
/test all

lukewarmtemp commented:
/test fcos-e2e

HuijingHei commented:
The CI failed about `Downloading https://kojipkgs.fedoraproject.org//packages/kernel/6.0.10/300.fc37/x86_64/kernel-6.0.10-300.fc37.x86_64.rpm...failed` when build https://github.com/coreos/layering-examples/blob/main/replace-kernel/Containerfile is not related to this PR.
